docs/using-keystone-webhook-authenticator-and-authorizer.md

```diff
@@ -64,9 +64,49 @@ This command creates the following entry in your ~/.kube/config
     auth-provider:
       name: openstack
 ```
-- Run `kubectl config set-context --cluster=kubernetes --user=openstackuser openstackuser@kubernetes`
+- Run `kubectl config set-context --cluster=<your cluster name> --user=openstackuser openstackuser@kubernetes`
```

Comment:
In this command you use "<your cluster name>" as cluster name, in the kubeconfig below you use "mycluster".
It's just a personal opinion, but I would prefer to use the same name in both examples.

Reply:
@gonzolino Thanks for the comments, do you mean using --cluster=mycluster at here?

Reply:
Yes that would be my suggestion. Or, alternatively, you could use "<your cluster name>" in the kubeconfig example.
Either way is ok for me, but I think it should be consistent.

Reply:
Ok, cool. I will do that. The current '--cluster=kuberntes' is very confusing, hence why I'd like to update it. Thanks. Will propose a new patch set.